app/controllers/admin/reports_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Admin
+  class ReportsController < ApplicationController
+
+    before_action :ensure_admin , only: [ :index ]
+
+    def index
+      @orders = Order.where(created_at: DateTime.current.beginning_of_day-5.day..DateTime.current.midnight)
+      unless report_params[:from_date].blank? && report_params[:to_date].blank?
```

Comment:
we dont need this, we have default values

Reply:
done